packages/amplify-graphql-index-transformer/src/__tests__/__snapshots__/amplify-graphql-primary-key-transformer.test.ts.snap

```diff
@@ -33,6 +33,8 @@ $util.qr($ctx.stash.put(\\"keys\\", $keys))
 #set( $lambdaInput.args.metadata = {} )
 #set( $lambdaInput.args.metadata.keys = [] )
 #set( $lambdaInput.args.metadata.nonScalarFields = [] )
+#set( $lambdaInput.args.metadata.fieldMap = {} )
+$util.qr($lambdaInput.args.metadata.fieldMap.putAll($util.defaultIfNull($context.stash.fieldMap, {})))
```

Comment:
Where is `context.stash.fieldMap` set? Can we add some snapshot to verify & test its datastructure?

Reply:
It's set in the `preDataLoad` for queries and `preUpdate` slot for mutations. These resolver slots are generated from the `refersTo` transformer. Will add a snapshot test there.